UI/data/themes/Yami.obt

```diff
@@ -1545,6 +1549,18 @@ MuteCheckBox::indicator:unchecked:disabled {
     image: url(theme:Dark/settings/audio.svg);
 }
 
+MuteCheckBox::indicator:indeterminate:hover {
+    image: url(theme:Dark/unassigned.svg);
+}
+
+MuteCheckBox::indicator:indeterminate:focus {
+    image: url(theme:Dark/unassigned.svg);
+}
+
+MuteCheckBox::indicator:indeterminate:disabled {
+    image: url(theme:Dark/unassigned.svg);
+}
+
```

Comment:
These lines are not necessary. The explicit hover/focus states are needed for checked/unchecked states because we set overrides for those for normal checkboxes

Reply:
Okay, I removed them.